### ISSUE TITLE
Alert when deleting a character sheet that has been imported to a token.

### ISSUE THREAD

**Issue by Aquaricat** [UnicornHeartClub/Issues#8]
When I delete a character sheet from My Vault, it does warn me that the action can't be undone, but it *doesn't* warn me if I'm deleting a sheet that's been attached to a token.  There should be some alert in this case just in case.  For example - maybe I have multiple sheets with the same name and character art for... whatever reason, or an old test sheet, and clicked on the wrong one to delete.  This is strictly a sanity check for DMs.

**Comment by tlackemann** [comment]
I don't believe this is necessary as token character sheets are copies of the character sheet itself. If you delete a character sheet, any "associated" tokens remain in-tact. All data is copied from sheet to token.

**Comment by Aquaricat** [comment]
It's not so much the fact that the token retains the sheet separately.
* I might have missed it, but currently there doesn't seem to be a way to recover a sheet from a token if the original is deleted.
* This is primarily informative: "BTW, if you forgot, you *have* created a token from this." rather than meant to prevent the deletion outright.

**Comment by Pjb518** [comment]
If there is no way to recover a sheet from a token, I'd be in favour of including this. I can see situations where people will regret deleting a sheet after the fact.
